src/getHumanAge.test.js

```diff
@@ -7,4 +7,65 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return an array`, () => {
+    expect(getHumanAge(0, 0))
+      .toBeInstanceOf(Array);
+  });
+
+  test(`should return an array of 0 and 0`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([
+        0, 0,
+      ]);
+  });
+
+  test(`should return an array of 14 and 14`, () => {
+    expect(getHumanAge(14, 14))
```

Comment:
try to write summary in more details. We need to understand what happened after working with this function. It is like test cases or bug report summary